src/amount.h

```diff
@@ -17,7 +17,7 @@ typedef int64_t CAmount;
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
 
-extern const std::string CURRENCY_UNIT;
```

Comment:
@gandrewstone there are 2 strings to move out the header file.  They were put there because bitcoin-cli has link failures arising from allowed-args.cpp otherwise.  Shall I move the currency unit and tor string to allowed-args.cpp?  Feels awkward but I don't see a better idea either.